src/server/worker/driver/driver_windows.go

```diff
@@ -0,0 +1,93 @@
+// +build windows
+
+package driver
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/pachyderm/pachyderm/src/client"
+	"github.com/pachyderm/pachyderm/src/server/worker/common"
+)
+
+// Note: these are stubs only meant for tests - the worker does not run on windows
```

Comment:
Ahh, I missed this last time through. I was going to ask about this because I was worried we might be pursuing that for some reason. Phew.